changelog.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Features
+
+* [3461](https://github.com/zeta-chain/node/pull/3461) - enable multiple confirmation count values in chain params
```

Comment:
consider modifying the changelog entry to be more accurate as we are adding a new ConfirmationParams object

Reply:
updated https://github.com/zeta-chain/node/pull/3461/commits/ea3f43b2cf2fc46c60459e7487ce0ddc60000194